src/utils/components/EvictionStrategy/EvictionStrategyModal.tsx

```diff
@@ -26,17 +29,30 @@ const EvictionStrategyModal: React.FC<EvictionStrategyModalProps> = ({
   vmi,
 }) => {
   const { t } = useKubevirtTranslation();
-  const [checked, setChecked] = React.useState<boolean>(
-    !!vm?.spec?.template?.spec?.evictionStrategy,
-  );
+  const isLiveMigrate =
+    vm?.spec?.template?.spec?.evictionStrategy === EVICTION_STRATEGIES.LiveMigrate;
+
+  const [checked, setChecked] = useState<boolean>(isLiveMigrate);
+  const [hyperConverge, hyperLoaded, hyperLoadingError] = useHyperConvergeConfiguration();
+
+  useEffect(() => {
+    if (vm?.spec?.template?.spec?.evictionStrategy || hyperLoadingError || !hyperLoaded) return;
+
+    if (hyperConverge?.spec?.evictionStrategy) {
+      setChecked(hyperConverge?.spec?.evictionStrategy === EVICTION_STRATEGIES.LiveMigrate);
+      return;
+    }
+
+    setChecked(EVICTION_STRATEGY_DEFAULT === EVICTION_STRATEGIES.LiveMigrate);
```

Comment:
?

Reply:
If the vm and the hyperconverge do not have a strategy, make it true. 
I can even use true directly